frontend/web/components/pages/EnvironmentSettingsPage.js

```diff
@@ -115,32 +115,10 @@ const EnvironmentSettingsPage = class extends Component {
 
   saveEnv = (e) => {
     e && e.preventDefault()
-    const { description, name } = this.state
-    if (ProjectStore.isSaving || !name) {
-      return
-    }
-    const has4EyesPermission = Utils.getPlansPermission('4_EYES')
 
     const env = _.find(ProjectStore.getEnvs(), {
       api_key: this.props.match.params.environmentId,
     })
-    AppActions.editEnv(
-      Object.assign({}, env, {
-        allow_client_traits: !!this.state.allow_client_traits,
-        banner_colour: this.state.banner_colour,
-        banner_text: this.state.banner_text,
-        description: description || env.description,
-        hide_disabled_flags: this.state.hide_disabled_flags,
-        hide_sensitive_data: !!this.state.hide_sensitive_data,
-        minimum_change_request_approvals: has4EyesPermission
-          ? this.state.minimum_change_request_approvals
-          : null,
-        name: name || env.name,
-        use_identity_composite_key_for_hashing:
-          !!this.state.use_identity_composite_key_for_hashing,
-        use_mv_v2_evaluation: !!this.state.use_mv_v2_evaluation,
-      }),
-    )
```

Comment:
It looks like removing these lines breakes the ability to update the environment name.
![image](https://github.com/Flagsmith/flagsmith/assets/41410593/7f69fe45-69e3-4585-9a6b-dbcccacd474b)